pt-BR/mail/chrome/messenger/preferences/privacy.dtd

```diff
@@ -1,41 +1,41 @@
 <!-- Junk Mail Controls -->
-<!ENTITY manualMark.label "When I mark messages as junk:">
+<!ENTITY manualMark.label "Quando eu marcar mensagens como spam:">
 <!ENTITY manualMark.accesskey "W">
-<!ENTITY manualMarkModeMove.label "Move them to the account's &quot;Junk&quot; folder">
+<!ENTITY manualMarkModeMove.label "Mover à pasta &quot;Spam&quot; da conta">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY manualMarkModeMove.label "Mover para a pasta &quot;Spam&quot; da conta">